README.rst

```diff
@@ -213,7 +213,7 @@ file for more info.
 .. _Auth0: https://auth0.com
 
 .. |pypi| image:: https://img.shields.io/pypi/v/auth0-python.svg?style=flat-square&label=latest%20version
-    :target: https://pypi.python.org/pypi/auth0-python
+    :target: https://pypi.org/project/auth0-python
```

Comment:
Link should include a trailing slash, like it does when redirected:

`https://pypi.org/project/auth0-python/`